cmd/crank/beta/xpkg/init.go

```diff
@@ -43,21 +43,40 @@ func WellKnownTemplates() map[string]string {
 
 // initCmd initializes a new package repository from a template repository.
 type initCmd struct {
-	Name     string `arg:"" help:"Name of the package to initialize."`
-	Template string `arg:"" help:"Template to initialize the package from."`
+	Name     string `arg:"" help:"The name of the new package to initialize."`
+	Template string `arg:"" help:"The template name or URL to use to initialize the new package."`
 
-	Directory string `short:"d" help:"Path of the directory to initialize." default:"." type:"path"`
+	Directory string `short:"d" default:"." type:"path" help:"The directory to initialize. It must be empty. It will be created if it doesn't exist."`
 }
 
 func (c *initCmd) Help() string {
+	tpl := `
+Crossplane can be extended using packages. A Crossplane package is sometimes
+called an xpkg. Crossplane supports configuration, provider and function
+packages. 
+
+A package is an opinionated OCI image that contains everything needed to extend
+Crossplane with new functionality. For example installing a provider package
+extends Crossplane with support for new kinds of managed resource (MR).
+
+This command initializes a directory that you can use to build a package. It
+uses a template to initialize the directory. It can use any Git repository as a
+template.
+
+You can specify either a full Git URL or a well-known name as a template. The
+following well-known template names are supported:
+
+%s
+
+See https://docs.crossplane.io/latest/concepts/packages for more information.
+`
```

Comment:
This renders as:

```
$ go run cmd/crank/main.go beta xpkg init --help
Usage: crossplane beta xpkg init <name> <template>

Initialize a new package from a template.

Crossplane can be extended using packages. A Crossplane package is sometimes
called an xpkg. Crossplane supports configuration, provider and function
packages.

A package is an opinionated OCI image that contains everything needed to extend
Crossplane with new functionality. For example installing a provider package
extends Crossplane with support for new kinds of managed resource (MR).

This command initializes a directory that you can use to build a package.
It uses a template to initialize the directory. It can use any Git repository as
a template.

You can specify either a full Git URL or a well-known name as a template.
The following well-known template names are supported:

  - provider-minimal (https://github.com/crossplane/provider-template)
  - provider-upjet (https://github.com/upbound/upjet-provider-template)
  - function-go (https://github.com/crossplane/function-template-go)

See https://docs.crossplane.io/latest/concepts/packages for more information.

Arguments:
  <name>        The name of the new package to initialize.
  <template>    The template name or URL to use to initialize the new package.

Flags:
  -h, --help             Show context-sensitive help.
      --verbose          Print verbose logging statements.
  -v, --version          Print version and quit.

  -d, --directory="."    The directory to initialize. It must be empty. It will
                         be created if it doesn't exist.
```